pgjdbc/src/main/java/org/postgresql/translation/ru.po

```diff
@@ -732,36 +745,38 @@ msgid ""
 "was created in.  The most common example of this is storing 8bit data in a "
 "SQL_ASCII database."
 msgstr ""
-"Найдены неверные символьные данные.  Причиной этого скорее всего являются "
-"хранимые данные содержащие символы не соответствующие набору символов базы.  "
+"Найдены неверные символьные данные. Причиной этого скорее всего являются "
+"хранимые данные содержащие символы не соответствующие набору символов базы. "
 "Типичным примером этого является хранение 8-битных данных в базе SQL_ASCII."
 
 # key: postgresql.call.noreturnval
 #: org/postgresql/jdbc/PgCallableStatement.java:85
 #: org/postgresql/jdbc/PgCallableStatement.java:95
 msgid "A CallableStatement was executed with nothing returned."
-msgstr ""
+msgstr "Команда CallableStatement не вернула результатов, хотя они ожидались."
 
 #: org/postgresql/jdbc/PgCallableStatement.java:106
 msgid "A CallableStatement was executed with an invalid number of parameters"
-msgstr ""
+msgstr "Количество колонок в результате CallableStatement не соответствует количеству OUT параметров."
 
 #: org/postgresql/jdbc/PgCallableStatement.java:144
 #, java-format
 msgid ""
 "A CallableStatement function was executed and the out parameter {0} was of "
 "type {1} however type {2} was registered."
 msgstr ""
+"Тип OUT параметра {0} {1} не соответствует тому, который использовался при регистрации: {2}."
 
 #: org/postgresql/jdbc/PgCallableStatement.java:206
 msgid ""
 "This statement does not declare an OUT parameter.  Use '{' ?= call ... '}' "
 "to declare one."
 msgstr ""
+"Для использования OUT параметров используйте синтаксис'{' ?= call ... '}'."
 
 #: org/postgresql/jdbc/PgCallableStatement.java:229
 msgid "wasNull cannot be call before fetching a result."
```

Comment:
I'd say the original message here is not proper English to begin with :) "... cannot be call**ed** ..."